### ISSUE TITLE
Rename testsys namespace and apiversion to something other than `testsys.bottlerocket.aws`

### ISSUE THREAD

**Issue by etungsten** [bottlerocket-os/bottlerocket-test-system#628]
We need to reconsider `testsys.bottlerocket.aws` as the CRD apiVersion string and `testsys-bottlerocket-aws` the default namespace since TestSys as a testing framework shouldn't be specific to AWS or Bottlerocket.

Ideas so far
- Plainly `testsys` for both
  - simple
  - but prone to naming collisions.
- `bottlerocket-test-system` for the namespace, `bottlerocket.testsys` for the API version prefix string
  - aligns well with `kube-system` and `eksa-system`
- ???

**Comment by webern** [comment]
TL;DR: I think `testsys` is fine.

The probability of a name collision seems low enough with `testsys` to me. I briefly searched for other projects using it and didn't see any. I also searched for "kubernetes namespace conventions"  and mostly found "don't use the default namespace". I'm not so sure that there should be an expectation that unrelated projects should be able to run on the same cluster without namespace conflicts anyway.

`brupop` is a different case altogether since `brupop` is meant to be run on a cluster that is, for sure, intended to run other unrelated workloads. I suppose this is true of utility daemons in general, but testsys is a pretty specific-use application.

**Comment by etungsten** [comment]
The CRD API group name needs to have at least one dot for it to be a valid domain.

Maybe  `testsys.aws` for the CRD API group? @webern what do you think?

**Comment by webern** [comment]
> The CRD API group name needs to have at least one dot for it to be a valid domain.
> 
> Maybe `testsys.aws` for the CRD API group? @webern what do you think?

I'm Ok with that as well. Should the CRD API group match the namespace?

**Comment by stmcginnis** [comment]
I think we should get the `aws` out of there. I know there was some chatter about this potentially being useful outside of AWS and even outside of Bottlerocket. So as an open source project, I don't think we should artificially tie this to AWS.

**Comment by etungsten** [comment]
> Should the CRD API group match the namespace?

No, it does not have to match.

> I think we should get the aws out of there.

Agreed, `testsys.com` should be neutral enough.
